cmd/vic-machine/configure/configure.go

```diff
@@ -85,6 +105,59 @@ func (c *Configure) Flags() []cli.Flag {
 			Destination: &c.upgrade,
 			Hidden:      true,
 		},
+		cli.BoolFlag{
+			Name:        "regenerate-certs",
```

Comment:
In general, we don't introduce any new options in configure command, instead we doing same processing logic with create.
In this case, we might still rely on what user specified, to make decision. If user specified tls-key/tls-cert, and reload the certificate and replace existing one, but if user specified all parameters to generate certificate, then regenerate.
anyway, regenerate is one lower priority feature, if hard to do in the above logic, we could drop that feature.